src/Expression.php

```diff
@@ -8,7 +8,7 @@
  * Creates new expression. Optionally specify a string - a piece
  * of SQL code that will become expression template and arguments.
  */
-class Expression implements \ArrayAccess, \IteratorAggregate
+class Expression implements Expressionable, \ArrayAccess, \IteratorAggregate
```

Comment:
`Expressionable` interface should imply a class with expression methods, not a getter for an expression